### PR TITLE
VZ-11405: Fix Golang version to 1.20.10 in VMO

### PR DESCRIPTION
**Pull request by SmritiOjha23**

### CODE REVIEW

File: docker-images/verrazzano-monitoring-operator/Dockerfile [hunk 1]
```diff
@@ -2,20 +2,9 @@
...
-RUN yum update -y \
-    && yum-config-manager --save --setopt=ol7_ociyum_config.skip_if_unavailable=true \
-    && yum install -y oracle-golang-release-el7 \
-    && yum-config-manager --disable ol7_developer_golang\* \
-    && yum-config-manager --enable ol7_developer_golang119 \
-    && yum -y install golang \
-    && yum clean all \
-    && go version
-
-ARG BUILDVERSION
```
Comment (bhatpmk) [inline docker-images/verrazzano-monitoring-operator/Dockerfile:16]: Any particular reason to remove the arguments ? For example, BUILDVERSION is used in line no. 15.

File: docker-images/verrazzano-monitoring-operator/Dockerfile [hunk 2]
```diff
@@ -2,20 +2,9 @@
...
-ARG BUILDDATE
-ARG EXTLDFLAGS
+FROM ghcr.io/oracle/oraclelinux:8-slim AS build_base
+RUN microdnf update -y --setopt=install_weak_deps=0 --setopt=tsflags=nodocs && \
+    microdnf install -y golang-1.20.10-1.module+el8.9.0+90068+08775a1f.x86_64
```
Comment (bhatpmk) [inline docker-images/verrazzano-monitoring-operator/Dockerfile:7]: Please clean the cache, by calling "microdnf clean all", followed by rm -rf /var/cache/yum

Comment (SmritiOjha23) [reply docker-images/verrazzano-monitoring-operator/Dockerfile:7]: okay sure

File: docker-images/verrazzano-monitoring-operator/Dockerfile (hunk 1 above)
Comment (SmritiOjha23) [reply docker-images/verrazzano-monitoring-operator/Dockerfile:16]: yes okay, I will remove it.

Comment (SmritiOjha23) [reply docker-images/verrazzano-monitoring-operator/Dockerfile:16]: yes okay, I will add it.

File: docker-images/verrazzano-monitoring-operator/Dockerfile [hunk 3]
```diff
@@ -32,11 +25,16 @@ RUN groupadd -r verrazzano-monitoring-operator && \
 
 FROM $FINAL_IMAGE AS final
 
+FROM ghcr.io/oracle/oraclelinux:8-slim
```
Comment (bhatpmk) [inline docker-images/verrazzano-monitoring-operator/Dockerfile:28]: May I know the reason for this change ?

**Review by bhatpmk — APPROVED** [review]

**Pull request merged**